assets/javascripts/discourse/initializers/extend-for-kolide.js

```diff
@@ -7,12 +7,17 @@ import bootbox from "bootbox";
 function initializeWithApi(api) {
   const currentUser = api.getCurrentUser();
 
-  function attachAssignButton(cooked) {
-    const buttons = cooked.querySelectorAll("a.kolide-assign") || [];
+  function attachButtons(cooked) {
```

Comment:
This does not really attach any buttons, but `registerButtonEvents` or something like that.

Reply:
No, I'm using the `decorateCookedElement` method to create buttons inside the post content.